.gitignore

```diff
@@ -45,9 +45,10 @@ cmd/kas-fleet-manager/kas-fleet-manager
 # Ignore secret files
 /secrets/db.ca_cert
 /secrets/aws.*
-/secrets/vault.*
+/secrets/aws-secret-manager.*
```

Comment:
What would happen if we were to deploy in GCP?

Reply:
You mean create a Kafka cluster in gcp?
It should be fine - it is basically just like what we do for route53. The tls certs will be stored securely in aws secret manager.